parser.go

```diff
@@ -689,28 +692,12 @@ func (p *Parser) scanIgnoreWhitespace() (tok Token, lit string) {
 // unscan reverts the pointer on the buffer, callers should not unscan more then what was
 // previously read, or values larger then BUF_SIZE.
 func (p *Parser) unscan(n int) {
-	if p.buf.n+n >= BUF_SIZE {
+	if p.buf.n+n >= BUFSIZE {
 		panic("size exceeds buffer")
 	}
 	p.buf.n += n
 }
 
-func (p *Parser) unscanIgnoreWhitespace(n int) error {
```

Comment:
right, we never use that. There are some comments that mention this method. I can make a follow up to remove those confusing comments.